index.rst

```diff
@@ -13,13 +13,93 @@
+This document describes the concept of operations for running the Auxiliary Telescope in Survey Mode, which will become the primary operational mode for AuxTel. 
+The Survey Mode utilizes semi-autonomous, scheduler-driven operations and the roles, responsibilities, and expectations for this mode are defined. 
 
-Add content here
-================
+Guidelines for Operations
+=========================
 
-Add content here.
-See the `reStructuredText Style Guide <https://developer.lsst.io/restructuredtext/style.html>`__ to learn how to create sections, links, images, tables, equations, and more.
+Survey mode is intended to be the primary mode for Auxiliary Telescope (AuxTel) operations. 
+It is a passive mode of operations, with nighttime observations driven by the Scheduler with minimal staff intervention. 
+The central tenant of operations in this mode is that the AuxTel operations are secondary to any responsibilities the summit staff may have related to Simonyi Survey Telescope operations. 
+
+This results in the following guidelines for operations in survey mode: 
+
+- The observing staff are not be required to actively monitor or report on incoming data quality.
+- Logging will be minimal; limited to short narrative statements needed to report on changes during the night or provide context for faults and recovery attempts.
+- Faults will be recovered when observing staff have available time, and the AuxTel may remain in Standby state during the night for extended periods of time. 
+- Remote support will be available for fault resolution on an on-call basis. 
+- Any faults which put hardware at risk will be result in closing the AuxTel early for the night.
+- The decision for closing the AuxTel early is to be made by observing staff on summit, no additional approval is required.
+
+Daytime activities
+==================
+
+Daytime Checkouts
+-----------------
+Daytime checkouts should be run once per week, preferable on Monday afternoons. 
+They should also be run following any major software updates (such as a new cycle release), or hardware changes and fault recoveries.
+They can be performed any time during the day when the system is available. 
+
+Afternoon Calibrations and Venting
+----------------------------------
+Afternoon calibrations and venting, including opening the vent gates and enabling the fan, will be performed daily by the daytime observing specialist on a best-effort basis. 
+That is to say, if daytime engineering work within AuxTel prevents us from starting calibrations and venting before 16:00, these tasks can be skipped at the discretion of the daytime observing specialist.
+
+Nighttime Activities
+====================
+
+Nighttime Operations
+--------------------
+The AuxTel should be entirely scheduler-driven at night while in survey mode. 
+A single scheduler configuration should be provided each night to prevent detailed night plans requiring active observing support.
```

Comment:
suggest replacing "prevent detailed night plans" with "minimize the need for detailed night(ly?) plans"